docs/_tutorials/persistence-with-queues.md

```diff
@@ -119,11 +120,7 @@ To send a message, you must still create a message. The difference from sending
 
 ```java
 TextMessage message = session.createTextMessage("Hello world Queues!");
-producer.send(queue,
-              message,
-              DeliveryMode.PERSISTENT,
-              Message.DEFAULT_PRIORITY,
-              Message.DEFAULT_TIME_TO_LIVE);
+messageProducer.send(message, DeliveryMode.PERSISTENT, Message.DEFAULT_PRIORITY, Message.DEFAULT_TIME_TO_LIVE);
```

Comment:
Can you revert to passing the destination in the call to send(). I believe it is more clear for the samples and matches the discussion in the associated tutorials.